src/main/java/com/dongtronic/diabot/commands/nightscout/NightscoutCommand.kt

```diff
@@ -178,8 +182,8 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand(category, nu
         }
     }
 
-    private fun buildResponse(dto: NightscoutDTO, avatarUrl: String?, builder: EmbedBuilder) {
-        builder.setTitle(dto.title)
+    private fun buildResponse(dto: NightscoutDTO, avatarUrl: String?, displayOpts: String?, builder: EmbedBuilder) {
+        if(displayOpts!!.contains("trend")) builder.setTitle(dto.title)
```

Comment:
Instead of asserting not null (`!!`) which I don't think is always the case, add a new `if` before the checks that creates an empty string when it's null.

something like this

```kotlin
if (displayOpts == null) {
    displayOpts = ""
}
```